javascript-data-model/order-history.js

```diff
@@ -0,0 +1,112 @@
+var orderHistory = [
+  [
+    {
+      orderPlaced: 'August 4, 2020',
+      total: '$34.00',
+      shipTo: 'JS Masher',
+      order: [
+        {
+          orderNumber: '114-3941689-8772232'
+        },
+        'Order Details',
+        'Invoice'
+      ]
+    },
+    {
+      deliveredDate: 'Aug 8, 2020',
+      deliveryUpdate: '',
+      product: [
+        {
+          title: 'JavaScript for impatient programmers',
+          additionalInfo: 'Rauschmayer, Dr. Axel',
+          returnWindow: 'Return window closed on Sep 7, 2020',
+          price: '$31.55'
+        }
+      ]
+    }
+  ],
```

Comment:
Why an array of two objects for each order? i would make one object, and maybe give that object a property named `orderDetails` and another property named `products` to achieve the same purpose, but with better description of the data.